gm4_sweethearts/data/sweethearts/functions/sneaking.mcfunction

```diff
@@ -0,0 +1,4 @@
+# @s = sneaking player holding poppy
```

Comment:
Requires a comment showing where this function is called from (main?)

Reply:
The function states that in line 2

Reply:
This wasn't fixed when Sparks reviewed @blue

Reply:
Oh, sry. Blame me then!